### PR TITLE
Added pagination to document records table

### PR DESCRIPTION
**Pull request by flutistar**
*Issue #:* /bcgov/entity#23262

*Description of changes:*
- Add pagination to document records table.

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
Looks good to me.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://documents-ui-dev--pr-62-gfmokssh.web.app

File: document-service/documents-ui/src/interfaces/table-inferface.ts [hunk 1]
```diff
@@ -1,7 +0,0 @@
-// Define a type for the Table Column
-export interface TableColumnIF {
-  key: string
-  label: string
-  tooltipText?: string
-  sortable?: boolean
-}
```
Comment (cameron-eyds) [inline document-service/documents-ui/src/interfaces/table-inferface.ts:7]: Obsolete?

File: document-service/documents-ui/src/utils/documentRecords.ts [hunk 2]
```diff
@@ -13,4 +13,6 @@ export function debounce(func, timeout = 200) {
       func(...args);
     }, timeout);
   };
-}
\ No newline at end of file
+}
+
+export const pageSize = 100
```
Comment (cameron-eyds) [inline document-service/documents-ui/src/utils/documentRecords.ts:18]: Small thing but maybe put the const at the top of the file with a short comment describing it's purpose

**Review by cameron-eyds — APPROVED** [review]
Looks great, nice job!

Just a few minor comments/questions

File: document-service/documents-ui/src/composables/useDocuments.ts [hunk 3]
```diff
@@ -111,38 +113,39 @@ export const useDocuments = () => {
         }
         if (documentURL) {
           existingDoc.documentUrls.push(documentURL)
+        } else {
+          // TODO: Remove Remove google.com once document URL is coming
+          existingDoc.documentUrls.push("https://google.com");
```
[the remark this answers is not in the document]
Comment (flutistar) [reply document-service/documents-ui/src/composables/useDocuments.ts:118]: image

I suppose download link will be added by another ticket so I added a dummy URL for test purpose.

File: document-service/documents-ui/src/interfaces/table-inferface.ts (hunk 1 above)
Comment (flutistar) [reply document-service/documents-ui/src/interfaces/table-inferface.ts:7]: Yep

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://documents-ui-dev--pr-62-gfmokssh.web.app

File: document-service/documents-ui/src/composables/useDocuments.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (cameron-eyds) [reply document-service/documents-ui/src/composables/useDocuments.ts:118]: Sounds good!

**Review by cameron-eyds — APPROVED** [review]
Looks good!

**Review by doug-lovett — APPROVED** [review]
Looks great.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://documents-ui-dev--pr-62-gfmokssh.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://documents-ui-dev--pr-62-gfmokssh.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://documents-ui-dev--pr-62-gfmokssh.web.app

**Pull request merged**